src/pages/VaultsPage/DepositModal.tsx

```diff
@@ -33,12 +33,18 @@ export default function DepositModal({
   const [max, setMax] = useState<any>();
   const [submitting, setSubmitting] = useState(false);
   const contracts = useContracts();
-  (window as any).contracts = contracts;
```

Comment:
Oops--this was just for debugging and can be removed entirely

Reply:
Ok I deleted the related useEffect